app/views/auth/login.blade.php

```diff
@@ -6,7 +6,7 @@
             <div class="welcome-logo">
                 <img class="logo" height="50" src="{{ url('img/cachet-logo.svg') }}" alt="Cachet"/>
             </div>
-            {{ Form::open() }}
+            {{ Form::open(['url' => '/auth/login']) }}
```

Comment:
Yeh, named routes are best. I'm also saying this for things like linking to images too.